### PR TITLE
Allow Api Tokens to access User Admin actions

### PR DESCRIPTION
**Pull request by topocount**
User management via circle API keys is needed if we want to enable this functionality via the discord bot, while simultaneously allowing circle admins to retain transparent, fine-grained permissioning for the bot.

### CODE REVIEW

File: hasura/metadata/actions.yaml [hunk 1]
```diff
@@ -111,6 +112,7 @@ actions:
         - name: verification_key
           value_from_env: HASURA_EVENT_SECRET
     permissions:
+      - role: api-user
```
Comment (karelianpie) [inline hasura/metadata/actions.yaml:115]: Unless for other api users, for the bot we don't need the `deleteCircle`

Comment (topocount) [reply hasura/metadata/actions.yaml:115]: yeah, trying to see if we should activate others here.

**Review by karelianpie — APPROVED** [review]

**Comment by topocount** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1945** <a href="https://app.graphite.dev/github/pr/coordinape/coordinape/1945" target="_blank">Graphite</a>  👈
    * **PR #1955** <a href="https://app.graphite.dev/github/pr/coordinape/coordinape/1955" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/coordinape/coordinape/1945?utm_source=stack-comment).

File: hasura/schema/user/schema.graphql [hunk 2]
```diff
@@ -254,32 +255,6 @@ type GenerateApiKeyResponse {
   hash: String!
 }
 
-type GuildAdmin {
-  address: String!
-}
-
-input GuildInfoInput {
```
Comment (crabsinger) [inline hasura/schema/user/schema.graphql:261]: I think we don't want these to be removed...

@CryptoGraffe we want this for users too?

Comment (topocount) [reply hasura/schema/user/schema.graphql:261]: @CryptoGraffe It looks like a perms issue is preventing these from coming through on the user level.

**Review by CryptoGraffe — APPROVED** [review]

**Pull request merged**